onchange_helper/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============
+Onchange Helper
+===============
+
+This is a technical module. Its goal is to ease the play of onchange method directly called from Python code.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* depend on this module
+* call `yourmodel.play_onchanges(values, ['field'])`
```

Comment:
this not correct actually, as the only model considered by this module is `ir.rule` (see my comment below)

Reply:
Should be fixed now, thank you!